src/wp-includes/media.php

```diff
@@ -5506,7 +5641,7 @@ function wp_get_loading_attr_default( $context ) {
 	}
 
 	// Special handling for programmatically created image tags.
-	if ( ( 'the_post_thumbnail' === $context || 'wp_get_attachment_image' === $context ) ) {
+	if ( 'the_post_thumbnail' === $context || 'wp_get_attachment_image' === $context ) {
```

Comment:
This change seems a little unrelated. Do we need it?

Reply:
@spacedmonkey I felt it didn't make much difference. Since the function is now deprecated there is hardly going the be any change in future, so did it in this ticket itself.